workflow-bot-app/src/main/java/com/symphony/bdk/workflow/WorkflowValidator.java

```diff
@@ -15,10 +15,6 @@
 public class WorkflowValidator {
 
   public static void validateFirstActivity(BaseActivity activity, Event event, String workflowId) {
-    if (activity.getIfCondition() != null) {
```

Comment:
I just noticed we don't prohibit "allOf" in this check. Actually when the first activity has a "allOf", the next activities will never be executed. (I was expecting new instance to start but the first one to have allOf events fulfilled and move forward, but it's worst).

Reply:
i will test this case to validate the behavior.